vhost_rs/src/vhost_user/master.rs

```diff
@@ -603,7 +603,7 @@ impl MasterInternal {
     #[inline]
     fn new_request_header(request: MasterReq, size: u32) -> VhostUserMsgHeader<MasterReq> {
         // TODO: handle NEED_REPLY flag
-        VhostUserMsgHeader::new(request, 0, size)
+        VhostUserMsgHeader::new(request, 0x1, size)
```

Comment:
Please change the commit message here, as the point is not about what QEMU is doing, but really what we actually support.

Reply:
Yes, thanks, i will change it.